static/app/components/repositoryProjectPathConfigForm.tsx

```diff
@@ -35,8 +47,34 @@ function RepositoryProjectPathConfigForm({
   projects,
   repos,
 }: Props) {
-  const orgSlug = organization.slug;
+  const api = useApi();
+  const formRef = useRef(new FormModel());
```

Comment:
uhg. I think I was using useState for this to avoid a new object being constructed every render